src/main/java/net/revelc/code/zookeeper/maven/plugin/StartZooKeeperMojo.java

```diff
@@ -37,7 +37,7 @@
 @Mojo(name = "start", defaultPhase = LifecyclePhase.PRE_INTEGRATION_TEST, threadSafe = true)
 public class StartZooKeeperMojo extends AbstractZooKeeperMojo {
 
-  @Parameter(defaultValue = "${project.build.directory}/zmp", readonly = true)
+  @Parameter(defaultValue = "${project.build.directory}/zmp", property = "zmp.directory")
```

Comment:
Thanks! I believe I've addressed the comments for this parameter annotation. Let me know if I have not.